src/app/components/FeedFilterElement/FeedFilterElement.tsx

```diff
@@ -0,0 +1,24 @@
+import classes from './FeedFilterElement.module.css';
+
+type FeedFilterElementProps = {
+  children?: React.ReactNode;
+  active?: boolean;
+  onClick?: () => void;
+};
+
+function FeedFilterElement({
+  children,
+  active,
+  onClick,
+}: FeedFilterElementProps): JSX.Element {
+  return (
+    <div className={classes.container}>
+      <button className={classes.button} onClick={onClick}>
+        {children}
+      </button>
+      {active && <div className={classes.active}></div>}
+    </div>
```

Comment:
You can reduce it to a single button if you use `border-bottom` in CSS to display to `active` line.